config/dev

```diff
@@ -1,5 +1,11 @@
+instanceid = dev
+
 include Makefile
 
+export base_url = /${instanceid}
```

Comment:
Do we really use such base URLs?

I think we all use ``/`` in our dev instances, ditto for demos. By the way I don't think it would work (there are often relative URLs in the UI such as ``/images/12345``).

What about fixing ``export base_url = /`` once for all in ``config/default``?